sci-libs/composable-kernel/composable-kernel-6.3.0.ebuild

```diff
@@ -86,6 +87,17 @@ src_configure() {
 		)
 	fi
 
+	# It takes ~2Gb of RAM per build thread
+	local user_jobs=$(makeopts_jobs)
+	local free_memory_mb=$(free -m | awk '/Mem:/ {print $4}')
+	local max_jobs=$(( free_memory_mb / 2048 ))
+	max_jobs=$(( max_jobs < 1 ? 1 : max_jobs ))
+	local limited_jobs=$(( user_jobs < max_jobs ? user_jobs : max_jobs ))
+	if [[ "${max_jobs}" -lt "${user_jobs}" ]]; then
```

Comment:
Let's use `check-reqs.eclass`, please. If we really want to go this route, we should revive my https://github.com/gentoo/gentoo/pull/23311 instead.

Reply:
Updated to use `CHECKREQS_MEMORY` (copied from 6.4.1 to 6.3.0).